src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Link.js

```diff
@@ -74,6 +100,7 @@ export default class Link extends React.Component<FieldTypeProps<LinkValue>> {
                 enableAnchor={enableAnchor}
                 enableTarget={enableTarget}
                 enableTitle={enableTitle}
+                excludedTypes={excludedProviderTypes}
```

Comment:
Feels a little bit strange having both types and excludedTypes but I understand that LinkContainer has the registry and so is the only place to filter them out. So its fine for me.